25.05 Cloud release notes (#4485)
centreon/centreon-documentation: cloud/releases/cloud-release-notes.md

title: Centreon Cloud release notes
---

## July 29th, 2025

### Centreon IT Edition

**Centreon Monitoring Agent**

As the Centreon Monitoring Agent (CMA) is about to be generally available, enhancements in its configuration are now available for our Centreon Cloud customers.

As a reminder, CMA collects metrics and computes statuses on the servers it monitors, and sends them to Centreon. 
Centreon plugins as well as Nagios-compatible plugins can be used with this agent.
More information about configuration is available in our [official documentation](https://docs.centreon.com/pp/integrations/plugin-packs/getting-started/how-to-guides/cma/).

This version adds the following enhancements to agent communication configuration:
- A "no TLS" option was added for troubleshooting purposes (valid for a time-limited period).
- It is now possible to store certificates in sub-directories.

> With the changes introduced in communication schemes, if you are planning on using CMA,

> > it is important to update your poller to at a minimum v24.10.8 of the Centreon Collect package.

Author: ```suggestion
> it is important to update your poller to at least version 24.10.8 of the Centreon Collect package.
```

**Centreon Dashboards**

> The Metrics graph widget has been enhanced to add the name of the service and/or host when needed in order to more clearly identify metrics that have the same name.

Author: ```suggestion
The **Metrics graph** widget has been enhanced to add the name of the service and/or host when needed in order to more clearly identify metrics that have the same name.
```

**Resource status**

> It is now possible to export the current data being as filtered by Resource status in CSV format.

Author: ```suggestion
It is now possible to export the current data as filtered by Resource status in CSV format.
```

**Host group configuration**

The configuration page for host groups has been improved to provide a better experience with listing, adding, editing etc.

> ### Centreon Cloud Pro

Reviewer: Mettre plutôt Centreon Pro Edition

> ### Centreon Cloud Pro

Author: ```suggestion
### Centreon Pro Edition
```

> ### Centreon Business Edition

Author: ```suggestion
### Centreon Cloud Pro
```

**MAP**

The MAP widget is now operational when displayed in Dashboard public playlists.
An option to "Display icons" was added to map containers and resources. If checked, custom and state icons are displayed on the shape.

**Business Activities**

> A new Dashboard widget was added: "Business Activity status timeline". It displays the distribution of current statuses on a BA, as a chronological timeline for a given time period.

Author: ```suggestion
A new Dashboard widget was added: **Business Activity status timeline**. It displays the distribution of current statuses on a BA, as a chronological timeline for a given time period.
```

> In addition, a new, modern version of the BA monitoring page is now available that includes the BA tree, BA timeline, and lists BA KPIs.

Author: ```suggestion
In addition, a new, modern version of the BA monitoring page is now available. It includes the BA tree and the BA timeline, and lists BA KPIs.
```

As a reminder, CMA collects metrics and computes statuses on the servers it monitors, and sends them to Centreon. 
Centreon plugins as well as Nagios-compatible plugins can be used with this agent.
More information about configuration is available in our [official documentation](/pp/integrations/plugin-packs/getting-started/how-to-guides/cma/).

This version adds the following enhancements to agent communication configuration:
- A "no TLS" option was added for troubleshooting purposes (valid for a time-limited period).
- It is now possible to store certificates in sub-directories.
- It is now possible to generate a token for the CMA agent.

> With the changes introduced in communication schemes, if you are planning on using CMA,
> it is important to update your poller to at a minimum v24.10.8 of the Centreon Collect package.

**Centreon Dashboards**

The Metrics graph widget has been enhanced to add the name of the service and/or host when needed in order to more clearly identify metrics that have the same name.

**Resource status**

It is now possible to export the current data being as filtered by Resource status in CSV format.

> **Meta-services**

Reviewer 2: The meta-services doesn't exist in Cloud so no need to add it in RN
